### PR TITLE
Jon loading nav refactor

### PR DESCRIPTION
**Pull request by jonramm**
- Added loading spinners to ReturnBike and MapPage
- Title screen displayed before login (VERY generic, for now it just has our names and the app name, you can look at it for longer than three seconds by logging out then changing the second parameter in _setTimeout_ to a greater value)
- Won't let user check out bike if it's more than 100 meters away. That value is in the constants folder if we want to change it.

### CODE REVIEW

File: client/constants/distance.tsx [hunk 1]
```diff
@@ -1,5 +1,7 @@
-const BIKE_RADIUS = 5000;
+const BIKE_RADIUS = 3000;
+const MIN_BIKE_DISTANCE = 100;
```
Comment (schaefkr) [inline client/constants/distance.tsx:2]: Does MAX_BIKE_DISTANCE make more sense?

File: client/pages/BikeInfo.tsx [hunk 2]
```diff
@@ -73,14 +80,14 @@ const BikeInfo = ({route, navigation}) => {
         };
         Promise.all([patchBike(bike.bike_id, patchBikeParams), addRide(addRideParams)]).then(responses => {
             if (responses[0].status === 201 && responses[1].status === 201){
+                startTimer(); // If everything is groovy we start the ride timer here
```
Comment (schaefkr) [inline client/pages/BikeInfo.tsx:83]: sounds good

File: client/pages/MapPage.tsx [hunk 3]
```diff
@@ -32,8 +33,8 @@ const MapPage = ({navigation}) => {
         )
     } else {
         return (
-            <View style={styles.container}>
-                <Text style={styles.loading}>Loading...</Text>
+            <View>
```
Comment (schaefkr) [inline client/pages/MapPage.tsx:36]: nice

**Review by schaefkr — APPROVED** [review]
looks good

File: client/constants/distance.tsx (hunk 1 above)
Comment (jonramm) [reply client/constants/distance.tsx:2]: Yes, absolutely!

**Pull request merged**